lib/normalize_country.rb

```diff
@@ -36,52 +36,103 @@ def to_h(key, options = {})
       hash
     end
 
+    def extend_countries(*args)
+      args = args.flatten
+      return if args.empty?
+
+      if args.first.is_a?(Hash)
+        args.each { |mapping| add_country(mapping, true) }
+      else
+        load_from_yaml(args.first, true)
+      end
+    end
+
+    alias_method :add_countries, :extend_countries
+
+    def reset!
+      Countries.clear
+      @to = @formats = nil
+
+      load_from_yaml File.join(File.dirname(__FILE__), "normalize_country", "countries", "en.yml")
+    end
+
     private
+
     def country_for(name)
-      name = name.to_s.downcase.strip.squeeze(" ")
+      name = Country.format_id(name.to_s.downcase)
       return if name.empty?
       Countries[name.to_sym]
     end
+
+    def add_country(mapping, merge = false)
+      country = Country.new(mapping)
+
+      if merge
```

Comment:
Oh, I see. So for all practical this exists to handle the addition of `aliases` or `short`s?